accounts/models.py

```diff
@@ -30,10 +32,12 @@ def create_superuser(self, email, password, **kwargs):
 
 class User(AbstractBaseUser, PermissionsMixin):
     USERNAME_FIELD = 'email'
-
-    email = models.EmailField(unique=True)
+    display_name = models.TextField(unique=True, null=True, max_length=30)
+    email = models.EmailField(unique=True, max_length=75)
```

Comment:
Why not leave this at the default max of 254?
A commend from the django source:

    # max_length=254 to be compliant with RFCs 3696 and 5321